### DISCUSSION TITLE
Allow players that choose Tank and Healer rolls in heroic dungeons to not be saved to an ID.

### DISCUSSION THREAD

**Discussion by Plan414** [chromiecraft/chromiecraft#6627] [Suggestions]
With the lower population especially in the NA hours. I suggest allowing the players that choose Tank and Healer rolls to not get saved to the heroic dungeons so they can then queue again. I'm typing this suggestion as my DPS character has been waiting in a queue for 1hr 20mins while tanks are saying "Im Saved".

**Comment by UnePomme** [comment]
As long as badge drops from bosses only occur on the 1st run, this sounds like a great idea with little to no downside.

**Reply by Plan414** [reply]
Even if tanks and healers received badges every run, how would this be a bad thing? We are entering the release of TK where having gear is becoming monditory. This would help gear up alts and new players.

**Reply by TotallyNotMyAccount** [reply]
"Healers and tanks should drag my ass through dungeons, but they mustn't get rewards for doing it!"
Uhhh, what?

**Reply by UnePomme** [reply]
> Even if tanks and healers received badges every run, how would this be a bad thing? We are entering the release of TK where having gear is becoming monditory. This would help gear up alts and new players.

True. I should have said it more as a thought piece than a fact. I don't disagree with you at all; my only concern is around balance. Blizzard balanced heroics the way they did, and taking those bars down could have some unforeseen consequences.

I would love to run heroics all day as a tank. I could get Reins of the Raven Lord in a day or two. And as much as I'd love that, I do have to acknowledge to myself that it may be a little imbalanced.

Now on the other hand. Add more heroics; let's see heroic deadmines!

**Reply by balleny** [reply]
but dont you keep getting badges in wrath if you queue random dungeon and already got an id on every dungeon (at that point, it just ignores your id and tps you into a fresh dungeon with your group)

**Reply by UnePomme** [reply]
> but dont you keep getting badges in wrath if you queue random dungeon and already got an id on every dungeon (at that point, it just ignores your id and tps you into a fresh dungeon with your group)

This sounds like the most ideal solution, especially since it's already a feature in place by blizzard.

**Comment by NinesCC** [comment]
This has been an issue ever since the heroics released. Rewards were proposed back then to incentivize players to queue / play more, but were ignored. Queues have been terrible for NA even when they were fresh content, never mind today.
